src/app/frontend/replicaset/detail/info_component.js

```diff
@@ -34,6 +38,13 @@ export default class ReplicaSetInfoController {
   areDesiredPodsRunning() {
     return this.replicaSet.podInfo.running === this.replicaSet.podInfo.desired;
   }
+
+  getLogsHref() {
+    return this.state_.href(
+        logsStateName,
+        new LogsStateParams(
+            this.replicaSet.objectMeta.namespace, this.replicaSet.objectMeta.name, 'ReplicaSet'));
```

Comment:
Backend has resource kind names defined as lower case strings in `api/types.go`.  We should use the same convention here.